### DISCUSSION TITLE
is:open 不支持R4S了 ？好像没有R4S的固件提供下载！

### DISCUSSION THREAD

**Discussion by fkdown** [xiaomeng9597/Actions-iStoreOS-For-RK35XX#7] [General]
is:open 不支持R4S了 ？好像没有R4S的固件提供下载！

**Comment by xiaomeng9597** [comment]
大雕的源码仓库吗，还是istoreos，istoreos官方固件站不是有固件下载？https://fw.koolcenter.com/iStoreOS/r4s/
